components/scream/tests/rrtmgp/input.yaml

```diff
@@ -9,11 +9,12 @@ Atmosphere Processes:
   Process 0:
     Process Name: RRTMGP
     Grid: Physics
+    Can Initialize All Inputs: true
 
 Grids Manager:
   Type: Physics Only
   Reference Grid: Physics
   Physics Only:
-    Number of global columns:   10
-    Number of vertical levels: 128
+    Number of global columns:  128
+    Number of vertical levels: 42
```

Comment:
Is this right?

Reply:
For the purpose of the standalone test, I think yes; the input problem has 42 vertical levels, and is computed for 128 columns.